### ISSUE TITLE
nodejs jspb.Message.getFieldWithDefault is not a function

### ISSUE THREAD

**Issue by pcj** [pubref/rules_protobuf#20]
0e65d2afe3a282049151dccd17c770b13f20a848 causes the nodejs server to fail:

```sh
INFO: Running command line: bazel-bin/examples/helloworld/node/server
/private/var/tmp/_bazel_pcj/63330772b4917b139280caef8bb81867/execroot/rules_protobuf/bazel-out/local-fastbuild/bin/examples/helloworld/proto/lib/node_modules/examples-helloworld-proto-node/examples/helloworld/proto/helloworld_pb.js:169
  return /** @type {string} */ (jspb.Message.getFieldWithDefault(this, 1, ""));
                                             ^

TypeError: jspb.Message.getFieldWithDefault is not a function
    at proto.helloworld.HelloRequest.getName (/private/var/tmp/_bazel_pcj/63330772b4917b139280caef8bb81867/execroot/rules_protobuf/bazel-out/local-fastbuild/bin/examples/helloworld/proto/lib/node_modules/examples-helloworld-proto-node/examples/helloworld/proto/helloworld_pb.js:169:46)
    at Object.sayHello (/Users/pcj/github/rules_protobuf/examples/helloworld/node/greeter_server.js:13:44)
    at Object.wrapper [as func] (/private/var/tmp/_bazel_pcj/63330772b4917b139280caef8bb81867/external/npm_grpc/lib/node_modules/grpc/node_modules/lodash/index.js:3095:19)
    at /private/var/tmp/_bazel_pcj/63330772b4917b139280caef8bb81867/external/npm_grpc/lib/node_modules/grpc/src/node/src/server.js:485:13
```

**Comment by pcj** [comment]
May be related to https://github.com/google/protobuf/pull/2193

**Comment by pcj** [comment]
Closed by going back to 3.0.0 in 0e65d2afe3a282049151dccd17c770b13f20a848.
